dev/MRTCore/packaging/MrtCore.props

```diff
@@ -0,0 +1,32 @@
+<!--
+  Copyright (c) Microsoft Corporation. Licensed under the MIT License
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <_ReunionImageGlobs>**/*.png;**/*.bmp;**/*.jpg;**/*.dds;**/*.tif;**/*.tga;**/*.gif</_ReunionImageGlobs>
+  </PropertyGroup>
+
+  <!--
+    By default, add the .RESW files in the project to PRIResource and the image files in the project to Content.
+    That's needed for the strings in the .RESW files and the image files to be indexed during PRI generation.
+    The app dev can opt out a file by changing the Build Action file property to None in Visual Studio, or
+    by manually modifying the CSPROJ file.
+
+    The implementation here is the same as the .NET SDK's implementation for default file inclusion.
+    See https://docs.microsoft.com/en-us/dotnet/core/project-sdk/overview#default-includes-and-excludes.
+
+    Yes, DefaultItemExcludes, DefaultExcludesInProjectFolder, EnableDefaultItems, and other properties here are
+    defined in a .NET SDK file that's imported after this one, but ItemGroup definitions are evaluated in a later pass.
+    See https://docs.microsoft.com/en-us/visualstudio/msbuild/build-process-overview?view=vs-2019#evaluation-phase  
+  -->
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp' and '$(EnableDefaultItems)'=='true' and '$(EnableDefaultContentItems)'!='false'">
+    <None Remove="$(_ReunionImageGlobs)" />
+    <Content Include="$(_ReunionImageGlobs)" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp' and '$(EnableDefaultItems)'=='true' and '$(EnableDefaultPRIResourceItems)'!='false'">
+    <None Remove="**/*.resw" />
+    <PRIResource Include="**/*.resw" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
```

Comment:
I think this makes sense but it did give me pause for a second. Specifically, the idea that `$(EnableDefaultPRIResourceItems)` doesn't control whether the files get globbed, just what item they get globbed into, is a bit surprising. Is that preferable to always removing resw from None (when using your package) and only conditionally including it?

I am fine with either way but want to make sure you think about it explicitly.

Reply:
I think leaving resw files out of `None` would more closely match what we now do in the .NET SDK.  If you turn `EnableDefaultCompileItems` off, the source files will not be part of `None`.  Originally the behavior was different, but we changed it to address [some bugs](https://github.com/dotnet/sdk/issues/1157) in Visual Studio behavior.

So I would lean towards removing `*.resw` from `None` regardless of whether `EnableDefaultPRIResourceItems` is set.  It's up to you, but if you want to go the other way I'd suggest testing what happens if you set the property to false and then add a resw file to the project in Visual Studio, both via the new item template as well as add / existing item.

Reply:
Went with Daniel's suggestion to remove `*.resw` from `None` regardless of whether `EnableDefaultPRIResourceItems` is set.